src/zrc/api/auth.py

```diff
@@ -0,0 +1,9 @@
+from django.conf import settings
+
+from zds_client import ClientAuth
+
+ztc_auth = ClientAuth(
```

Comment:
Denk dat we dit moeten vervangen met een modelletje, zodat we dit makkelijk configureerbaar kunnen maken in de admin. Nice-to-have maar wellicht wel voor API-lab?

Reply:
Akkoord :+1:

Reply:
Zie #25